config/kube-app-manager/manager.yaml

```diff
@@ -26,7 +19,7 @@ spec:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: controller
```

Comment:
same here. kubeapp-controller ?

Reply:
the generated output is : `kube-app-controller`

Reply:
ack